cocotb/regression.py

```diff
@@ -33,6 +33,12 @@
 import sys
 import os
 import traceback
+import pdb
+
+if "COCOTB_PDB_ON_EXCEPTION" in os.environ:
+    _pdb_on_exception = True
+else:
+    _pdb_on_exception = False
```

Comment:
This environment variable needs to be documented alongside the other ones in https://github.com/cocotb/cocotb/blob/master/documentation/source/building.rst